backend/src/main/java/ca/bc/gov/hlth/mohums/exceptions/RestExceptionHandler.java

```diff
@@ -0,0 +1,20 @@
+package ca.bc.gov.hlth.mohums.exceptions;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
+import org.springframework.http.ResponseEntity;
+
+@ControllerAdvice
+public class RestExceptionHandler extends ResponseEntityExceptionHandler {
```

Comment:
Good point. I'll separate it. I wanted to include exception handling so that the same error response as in Keycloak API is returned when user is not found. But this error handling only applies to GET `"/user/{userId}"` - I started with this simple endpoint to test out JPA integration, instead of jumping to the more complex GET `"/users"` endpoint right away.